docs/registry/configure.md

```diff
@@ -247,10 +208,11 @@ For any version type other than **Patch (X.Y.Z)**, the module will upgrade as so
 If, for example, the module provides a motor component, and the motor is running, it will stop while the module upgrades.
 {{% /alert %}}
 
-### Use environment variables with a registry module
+{{< /expand >}}
+{{< expand "Use environment variables with a registry module" >}}
```

Comment:
This is interesting and should be covered in the module attributes table I think.

Reply:
I don't think all of this content can be condensed into the attributes table so I'm thinking I should add the attribute to the table but still have a section of some sort on this page for details

Reply:
you can add the expander underneath the table. But the table should include the field you'd use to add the variables